src/App.js

```diff
@@ -5,18 +5,21 @@ import Footer from "./components/Footer";
 import { BrowserRouter as Router } from "react-router-dom";
 import { Component } from "react";
 import Search from "./components/Search";
+
 class App extends Component {
   state = {
     moves: [],
   };
 
-  componentDidMount() {
+  componentDidMount = () => {
```

Comment:
This does not work!
componentDidMount() is Reacts life cycle method. You are now overriding it 🤣

Reply:
hahaha ooo noooh :D it was just complaining something about binding and I googled solutions and they said stackoverflow that changing it to arrow function could help, but obviously not in this case :D. 
Let me try again!